src/services/embeds/providers/provider.youtube.js

```diff
@@ -19,7 +19,9 @@ module.exports = function (app) {
     }
 
     normalizeURL (url) {
-      if (this.match && this.match[1]) {
+      if (this.match && this.match[1] && this.match[2]) {
+        url = `https://www.youtube.com/watch?v=${this.match[1]}?${this.match[2]}`;
```

Comment:
Should the second `?` not be a `&`?

Reply:
[Yup](https://developers.google.com/youtube/player_parameters#start), will change it on the next pull request along with the testings.

Btw, @roschaefer, let's do it. Ping me on the chat so we can talk timezones.

Reply:
Just made the PR with the changes. One thing to mention though, is that the Metadata that is returned, has the start time appended with `?start=`, instead of the `&start=`. Not sure why though. 

But if you take the JSON from the response of a request to "https://www.youtube.com/watch?v=QWU9YsxXPqw&start=332", the returned value of `body.url` is  "https://www.youtube.com/watch?v=QWU9YsxXPqw" while `body.video[0].url` is "https://www.youtube.com/embed/QWU9YsxXPqw?start=332".